### DISCUSSION TITLE
Does using Vanara with WinUI3 &/- WinRT make sense?

### DISCUSSION THREAD

**Discussion by tajbender** [dahall/Vanara#313] [Q&A] [answered]
Hello, @dahall,

Just a quick and dirty questing:

Does using Vanara with WinUI3 and WinRT make sense?

Are there any known issues?

Thanks and regards in advance,

tajbender

**Comment by dahall** [comment, marked as answer]
I don't see any reason why not. Both should be similar to using with UWP.
